ppl/zqd/search/search.go

```diff
@@ -80,9 +81,11 @@ func (s *SearchOp) Run(ctx context.Context, store storage.Storage, output Output
 	switch st := store.(type) {
 	case *archivestore.Storage:
 		return driver.MultiRun(ctx, d, s.query.Proc, zctx, st.MultiSource(), driver.MultiConfig{
-			Span:      s.query.Span,
-			StatsTick: statsTicker.C,
-			Order:     zbuf.OrderDesc,
+			Span:            s.query.Span,
+			StatsTick:       statsTicker.C,
+			Order:           zbuf.OrderDesc,
+			Parallelism:     s.workers,
+			DistributedExec: (s.workers > 0),
```

Comment:
If workers is removed from the api (as suggested above), then DistributedExec would be set based on the zqd personality.

Reply:
It is OK to request that a zqd root worker not use distributed execution. Leaving workers set to 0 will ensure that the root process executes the query directly.